core/trino-main/src/main/java/io/trino/sql/planner/RelationPlanner.java

```diff
@@ -291,6 +291,29 @@ public RelationPlan addRowFilters(Table node, RelationPlan plan, Function<Expres
         return new RelationPlan(planBuilder.getRoot(), plan.getScope(), plan.getFieldMappings(), outerContext);
     }
 
+    public RelationPlan addCheckConstraints(List<Expression> filters, Table node, RelationPlan plan, Function<Expression, Expression> predicateTransformation, Function<Table, Scope> accessControlScope)
```

Comment:
Similarly as above, the name `filters` is misleading. Rename it to `constraints`.